src/components/auth/Login.js

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from "react";
 import axios from 'axios';
 import { useDispatch } from 'react-redux';
 import { connect } from 'react-redux';
+import GoogleLogo from '../../img/google_icon.png';
```

Comment:
It'd be better if the SVG format of this icon was imported as it preserves quality better than PNGs.